FAQ.md

```diff
@@ -91,3 +91,14 @@ setenv load_script 'if test -e mmc 0:1 boot/boot.scr; then echo \"... booting fr
 setenv bootcmd 'run get_images; run set_bootargs; run load_script;booti \$kernel_addr \$ramfs_addr \$fdt_addr'
 saveenv
 ```
+**Q:** Seems all my mac addresses are the same across multiple boards. How do I fix this?
```

Comment:
Is this specific to Espressobin? If not, it should be in the Armbian section of this file.

Reply:
Its an Arabian bug. not exclusive to espresso bin.

Reply:
Shift up the doc in Armbian section :)

Reply:
Duu :P :+1: lol